jobrunr-kotlin-support/build.gradle

```diff
@@ -7,6 +7,8 @@ plugins {
 dependencies {
   api project(':core')
 
+  api 'org.koin:koin-core:2.2.2'
```

Comment:
Would this also work with a compileOnly dependency? That way, if users don't use Koin, they don't get this transitive dependency?

Reply:
Sorry, for some reason I didn't get the email notification that you had reviewed the PR. I think in that case it would maybe be better to split the koin job activator into a separate koin support plugin?